dspace-api/src/main/java/org/dspace/administer/CreateAdministrator.java

```diff
@@ -84,9 +105,21 @@ public static void main(String[] argv)
     				line.getOptionValue("f"), line.getOptionValue("l"),
     				line.getOptionValue("c"), line.getOptionValue("p"));
     	}
+    	else if(line.hasOption("h"))
```

Comment:
I would test for -h first.  It is unlikely, but not impossible, that a user would specify all of the options and then become uncertain, add -h to the end without deleting the others, and be surprised when the command created an administrator and gave no help.  (I have done this with other commands.)

Reply:
That's done.

Also changed header text to explain that all parameters need to be passed if prompting is to be avoided.